excellent/evaluator.go

```diff
@@ -50,9 +49,7 @@ func EvaluateTemplate(env utils.Environment, context types.XValue, template stri
 	// if we only have an identifier or an expression, evaluate it on its own
 	if nextTT == EOF {
 		switch tokenType {
-		case IDENTIFIER:
-			return ResolveValue(env, context, token), nil
-		case EXPRESSION:
+		case IDENTIFIER, EXPRESSION:
```

Comment:
Previously we've been parsing identifiers and expressions in subtly different ways - i.e. a whole identifier like `a.b.c` is passed to `ResolveValue` and it iterates through each property resolving them - but the expression `(a.b.c)` will be parsed by the visitor and `ResolveValue` will only be used to get the `b` of `a` and the `c` of `b`.

Consolidating these simplifies the code and means they have to play by the same parsing rules.